packages/instant/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "4.1.0",
+        "changes": [
+            {
+                "note": "Run Web3ProviderEngine without excess block polling",
+                "pr": 1695
+            }
+        ]
+    },
```

Comment:
@dekz please merge this with the previous, unpublished CHANGELOG entry.

Reply:
Instant hasn't had a published timestamp since verison 1.0.1. What's the way to determine the current unpublished CHANGELOG?